fungible-token/src/lib.rs

```diff
@@ -187,70 +199,13 @@ impl FungibleToken {
     }
 }
 
-#[derive(Debug, Decode, TypeInfo)]
-struct InitConfig {
-    name: String,
-    symbol: String,
-}
-
-#[derive(Debug, Decode, TypeInfo)]
-struct MintInput {
-    account: H256,
-    amount: u128,
-}
-
-#[derive(Debug, Decode, TypeInfo)]
-struct BurnInput {
-    account: H256,
-    amount: u128,
-}
-
-#[derive(Debug, Encode, Decode, TypeInfo)]
-struct ApproveData {
-    owner: H256,
-    spender: H256,
-    amount: u128,
-}
-
-#[derive(Debug, Decode, Encode, TypeInfo)]
-struct TransferData {
-    from: H256,
-    to: H256,
-    amount: u128,
-}
-
-#[derive(Debug, Decode, Encode, TypeInfo)]
-struct TransferFromData {
-    owner: H256,
-    from: H256,
-    to: H256,
-    amount: u128,
-}
-
-#[derive(Debug, Decode, TypeInfo)]
-enum Action {
-    Mint(MintInput),
-    Burn(BurnInput),
-    Transfer(TransferData),
-    TransferFrom(TransferFromData),
-    Approve(ApproveData),
-    IncreaseAllowance(ApproveData),
-    DecreaseAllowance(ApproveData),
-}
-
-#[derive(Debug, Encode, TypeInfo)]
-enum Event {
-    Transfer(TransferData),
-    Approval(ApproveData),
-}
-
 gstd::metadata! {
     title: "FungibleToken",
-    init:
-        input: InitConfig,
-    handle:
-        input: Action,
-        output: Event,
+        init:
+            input : InitConfig,
+        handle:
+            input : Action,
+            output : Event,
```

Comment:
Prefer indents like in structs:
```suggestion
    init:
        input: InitConfig,
    handle:
        input: Action,
        output: Event,
```